.python-version

```diff
@@ -1 +1 @@
-3.11
```

Comment:
**issue**: pro is going to be 3.11. i think that is why this is 11 instead of 12.

Reply:
OK, Pro-bro. ;) I removed this commit.